wgpu-core/src/command/clear.rs

```diff
@@ -305,23 +356,23 @@ pub(crate) fn collect_zero_buffer_copies_for_clear_texture(
         let max_rows_per_copy = max_rows_per_copy / format_desc.block_dimensions.1 as u32
             * format_desc.block_dimensions.1 as u32;
         assert!(max_rows_per_copy > 0, "Zero buffer size is too small to fill a single row of a texture with format {:?} and desc {:?}",
-                        texture_desc.format, texture_desc.size);
+                            texture_desc.format, texture_desc.size);
```

Comment:
cargo fmt is needed?

Reply:
cargo fmt doesn't seem to have an opinion on this oddly enough, must be a bug